compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -70,7 +74,9 @@ internal actual class ComposeWindow : UIViewController {
     @OverrideInit
     constructor(coder: NSCoder) : super(coder)
 
-    private val density: Density = Density(1f) //todo get and update density from UIKit Platform
+    private val density: Density
+        get() = Density(UIScreen.mainScreen.scale.toFloat())
```

Comment:
`mainScreen`  implies that there can be other screens. I am not sure, but theoretically now or in the future we can plug additional display to the iOS device (iPad + PC monitor or iPhone with 2 displays).

Can we get screen from the view itself? In Skiko with use [the other property:](https://github.com/JetBrains/skiko/blob/2ed9168d1231dd154d79fb47295ebdbcc8a0236f/skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkiaLayer.ios.kt#L31)

Reply:
Makes sesne. 
Let's use `layout.layout.contentScale` from Skiko.
Fixed

Reply:
Just a note:
Unlike `UIScreen.mainScreen.scale`, we can read `contentScaleFactor` + we also can set an arbitrary value to `uiView.contentScaleFactor` (setting to a lower value makes the UI look blurry). 
I doubt we would need such a feature, but in case users would like to use it (set a custom value), it makes sense for us to it as a source of truth.

Reply:
> it makes sense for us to it as a source of truth

Understand. We had a similar situation in Swing - we wrongly used Density from Compose instead of using Density from Swing. And when they doesn't match (user override LocalDensity), we had bugs.

Probably it is less dangerous with `contentScaleFactor`, but if we go to the path of using density from 2 sources, we should pick the source carefully, avoiding picking the wrong one.